multi_scope_acquisition.py

```diff
@@ -672,7 +673,16 @@ def run_acquisition_bmotion(hdf5_path, toml_path, config_path):
         except ValueError:
             print("Invalid input. Please enter a number.")
 
-    selected_key, selected_mg = motion_groups[selection - 1]
+    selected_key = selection
+    if selected_key not in motion_groups:
+        selected_key = f"{selected_key}"
```

Comment:
The motion group key can actually be an int or str.  Since the GUI never allows the user to define the key, it should always be an int.  This if-clause to check for a string version is just for safety.